### PR TITLE
doc(README): clarify humane-js styles dependency

### LINKED ISSUE #39: Notifications do not disappear, and cannot be clicked (to close) unless you include at least original css classes from humane
If supported, humane relies on css transitions to remove messages.  Therefore if you do not include the minimum necessary classes, the messages never disappear.  As the clickToClose functionality uses the same method to make the notification disappear, it also doesn't work.

### PR DESCRIPTION
**Pull request by doktordirk**
closes https://github.com/SpoonX/aurelia-notification/issues/39

---

This change is [Reviewable](https://reviewable.io/reviews/spoonx/aurelia-notification/41)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg?maxAge=2592000?style=plastic)](https://gitter.im/SpoonX/Dev)
 
-This library is an unofficial plugin for the [Aurelia](http://www.aurelia.io/) platform and contains a simple but fully configurable notification service using [humane-js](https://www.npmjs.com/package/humane-js). While all options of humane-js are available (instead of adding callbacks, promises are returned where applicable though), basic usage doesn't require more than adding your own css or selecting one of humane-js css files as specified below.
+This library is an unofficial plugin for the [Aurelia](http://www.aurelia.io/) platform and contains a simple but fully configurable notification service using [humane-js](http://wavded.github.io/humane-js/). While all options of humane-js are available (instead of adding callbacks, promises are returned where applicable though), basic usage doesn't require more than adding your own css or selecting one of humane-js css files as specified below.
```
Comment (RWOverdijk) [inline README.md:5]: Make up your mind on the url :p

Comment (doktordirk) [inline README.md:5]: i know, i know. 
but's actually on purpose. those are the appropriate links for each topic

**Comment by doktordirk** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

_[README.md, line 5 [r1]](https://reviewable.io:443/reviews/spoonx/aurelia-notification/41#-KIEWFjrB7X-ehT715Jn-r1-5:-KIEWcVo3PqY_XVewiWY:-499146009) ([raw file](https://github.com/spoonx/aurelia-notification/blob/924a9f14aca60115edff045e816ef7e2f06ab8f5/README.md#L5)):_
<details><summary>

<i>Previously, doktordirk (doktordirk) wrote…</i></summary>



> i know, i know. 
> 
> but's actually on purpose. those are the appropriate links for each topic
> </details>
> Done.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/spoonx/aurelia-notification/41)_

**Comment by RWOverdijk** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/spoonx/aurelia-notification/41)_

**Comment by doktordirk** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/spoonx/aurelia-notification/41)_

**Pull request merged**